FWCore/Integration/test/unit_test_outputs/testGetBy1.log

```diff
@@ -132,6 +133,9 @@ ModuleCallingContext state = Running
 ++++ finished: begin stream for module: stream = 0 label = 'intProducerU' id = 12
 ++++ starting: begin stream for module: stream = 0 label = 'intVectorProducer' id = 13
 ++++ finished: begin stream for module: stream = 0 label = 'intVectorProducer' id = 13
+++ finished: begin stream 0
+++ starting: begin stream 0
+++ finished: begin stream 0
```

Comment:
Why are there two messages for `{starting,finished}: begin {job,stream 0}`? Is there a SubProcess?

Reply:
Yes.